alkira/resource_alkira_policy_prefix_list.go

```diff
@@ -220,10 +237,11 @@ func generatePolicyPrefixListRequest(d *schema.ResourceData, m interface{}) (*al
 	}
 
 	list := &alkira.PolicyPrefixList{
-		Description:  d.Get("description").(string),
-		Name:         d.Get("name").(string),
-		Prefixes:     convertTypeSetToStringList(d.Get("prefixes").(*schema.Set)),
-		PrefixRanges: prefixRanges,
+		Description:   d.Get("description").(string),
+		Name:          d.Get("name").(string),
+		Prefixes:      extractPrefixesFromSchema(d),
```

Comment:
I think you could use a single function to do this and return `prefixes` and `prefixDetails`. It will be cleaner. Function name could be like this: `expandPrefixes` as our convention to expand a block in TF, that returns both struct...

so

```
prefixes, prefixDetails, err := expandPrefixes()

if err != nil...
```